node-fs-writefile/cp.js

```diff
@@ -0,0 +1,15 @@
+import { readFile, writeFile } from 'node:fs/promises';
+async function copy() {
+  try {
+    const name = process.argv.slice(2);
+    const newFile = name.slice(1);
+    const fileTxt = await readFile(`${name}`, 'utf8');
+    const promise = await writeFile(`${newFile}`, `${fileTxt}`);
+    return promise;
```

Comment:
This isn't ever used so there's no need to return it. Also, it is not a Promise, it is the resolved value of the promise returned by writeFile.